### PR TITLE
Feat/attributesets

### PR DESCRIPTION
**Pull request by RossMurr4y**
## Description

Contributes to https://github.com/hamlet-io/engine/pull/1498
* Remove reference data for metaparameters
* Add reference data for attributesets

## Target Audience

- [x] hamlet users
- [x] hamlet framework developers (incl. plugins)

#### Documentation
- [ ] Spelling / Syntax correction only
- [x] Refactor (Documentation overhaul, or revising after changes to hamlet)
- [ ] New feature (Documenting something new)
#### Site Design
- [ ] Refactor (No new or removed content.)
- [ ] New Features & Content
- [ ] Docusaurus / Plugin version update.

## Followup Actions

- [x] None

## Checklist:

- [x] I have tested the changes locally - see [README.md](https://github.com/hamlet-io/docs/blob/master/README.md)
- [x] I have added appropriate labels to this PR.
- [x] I have added this to the `hamlet roadmap` project.

### CODE REVIEW

File: docs/foundations/anatomy.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ The `Account` is linked to the `Tenant` CMDB and inherits its configuration. A `
 
 ## Product CMDB
 
-A `Product` contains multipe `Solutions` which can be deployed into `Environments` to manage the full lifecycle of the `Product`. The `Solution` defines the `Components` and supporting configuration (`ReferenceData` and `MetaParameters`) that configure your `Product`.
+A `Product` contains multipe `Solutions` which can be deployed into `Environments` to manage the full lifecycle of the `Product`. The `Solution` defines the `Components` and supporting configuration (`ReferenceData` and `AttributeSets`) that configure your `Product`.
```
Comment (roleyfoley) [inline docs/foundations/anatomy.md:27]: I don't think a user would put an AttirbuteSet into a solution. They are more internal development reference to make life easier for development aren't they?

File: docs/foundations/anatomy.md [hunk 2]
```diff
@@ -75,6 +75,6 @@ Each of the above levels can then be further broken down as required by director
...
+Within a CMDB, any file under the path 'config/solutionsv2' contributes to a Hamlet `Solution` and can contain [`Components`](https://hamlet.io/reference/components), [`ReferenceData`](https://hamlet.io/reference/data) and [`AttributeSets`](https://hamlet.io/reference/attributesets) data types. Configuration is applied in a top-down manner at the CMDB-level (`Tenant > Account > Product`) and then again at the `Solution`-level (`Environment > Segment > Deployment Unit`). New content is combined with existing, whilst existing content will be overwritten if re-implemented by a more specific scope. This allows for the most effecient implementation for a given use-case.
```
Comment (roleyfoley) [inline docs/foundations/anatomy.md:78]: Same here re: Attribute Sets

File: docs/foundations/profiles.md [hunk 3]
```diff
@@ -43,10 +43,10 @@ Deployment and Policy profile types have two filters - Deployment Mode and Compo
 
 A "*" (any) filter can be used for either filter to represent matching any mode or component type. This is not a wildcard however, and cannot be used for pattern matching.
 
-## The Profiles Meta Parameter
-Profiles are assigned to a scope though the `Profiles` metaparameter object, containing key/value pairs of the profile types and names. The metaparameter object is available to all scopes, however only the Deployment and Policy profile types are available for assignment above the Component level. To implement a Component-level profile at a higher scope, they should be associated with either a Deployment or Policy profile.
+## The Profiles ReferenceData Type
+Profiles are assigned to a scope though the `Profiles` Reference Data object, containing key/value pairs of the profile types and names. The Reference Data object is available to all scopes, however only the Deployment and Policy profile types are available for assignment above the Component level. To implement a Component-level profile at a higher scope, they should be associated with either a Deployment or Policy profile.
```
Comment (roleyfoley) [inline docs/foundations/profiles.md:47]:
```suggestion
Profiles are assigned to a scope though the `Profiles` Reference Data object, containing key/value pairs of the profile types and names. The Profiles object is available to all scopes, however only the Deployment and Policy profile types are available for assignment above the Component level. To implement a Component-level profile at a higher scope, they should be associated with either a Deployment or Policy profile.
```

**Review by roleyfoley — CHANGES_REQUESTED** [review]

File: docs/foundations/anatomy.md (hunk 1 above)
Comment (RossMurr4y) [reply docs/foundations/anatomy.md]: Yep you're right. Updated here in https://github.com/hamlet-io/docs/pull/77/commits/f79ad97ae2cd25ded3c2f5b1c69009465c8cfe30

File: docs/foundations/anatomy.md (hunk 2 above)
Comment (RossMurr4y) [reply docs/foundations/anatomy.md]: Updated here too https://github.com/hamlet-io/docs/pull/77/commits/6e823cfc759866a19c08ed795077fe81e1f61820

**Pull request merged**